src/documents/tutorial-gettingStartedWithInfusion/DefineANamespaceAndCreateAClosure.md

```diff
@@ -43,13 +43,13 @@ So what does this look like in general?
 })(jQuery, fluid_1_5);
 ```
 
-You might like to think of the [`fluid.registerNamespace`](https://github.com/fluid-project/infusion/blob/infusion-1.5/src/framework/core/js/Fluid.js#L957-L966) call as equivalent to a line such as `var mynamespace = mynamespace || {};` written at the global scope. It is less cumbersome and more expressive of your intention, as well as easily allowing you to declare nested namespaces in one definition. Use this framework utility unless your requirements are very sophisticated (that is, you are writing a framework which you expect to be version-managed independently of both jQuery and fluid - in which case you should supply your framework's global object as an argument to the overall file closure.
+You might like to think of the [`fluid.registerNamespace`](https://github.com/fluid-project/infusion/blob/infusion-1.5/src/framework/core/js/Fluid.js#L957-L966) call as equivalent to a line such as `var mynamespace = mynamespace || {};` written at the global scope. It is less cumbersome and more expressive of your intention, as well as easily allowing you to declare nested namespaces in one definition. Use this framework utility unless your requirements are very sophisticated (that is, you are writing a framework which you expect to be version-managed independently of both jQuery and fluid - in which case you should supply your framework's global object as an argument to the overall file closure).
```

Comment:
Do we have a strategy for the api links into the source code for code from 'master'. Currently these links are pointing at 1.5. If we point at master though, we would have to be careful about them breaking when code is modified.